### DISCUSSION TITLE
Have Github Pro (Student Dev Pack) but copilot says require payment

### DISCUSSION THREAD

**Discussion by ronakice** [community/community#43640] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hey,

So I have Github Pro from the Student Developer Pack and I've regularly been using Copilot. But today, it says I require to make a payment to continue using copilot despite my account being covered still by both Github Pro and the Student Developer Pack.

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by otkrickey** [comment]
Solved: #43634
